tutorials/cloud-iot-enviro-board-workshop/enviro-device/core.py

```diff
@@ -167,7 +167,7 @@ def publish_message(self, message):
 
     def register_message_callbacks(self, callbacks):
         """
-        Specifies functions to call upon various MQTT pub-sub messages.
+        Specifies functions to call upon various MQTT Cloud Pub/Sub messages.
```

Comment:
This is MQTT pub-sub, is not related to the Cloud Pub/Sub

Reply:
Ah, sorry about that. Thanks for catching it.